app/assets/javascripts/discourse/models/topic.js.es6

```diff
@@ -283,6 +283,15 @@ const Topic = RestModel.extend({
     });
     return Discourse.ajax("/t/" + this.get('id') + "/recover", { type: 'PUT' });
   },
+  
+    // Change Timestamp
+  changeTimeStamp(post, dateTime) {
+      return Discourse.ajax("/topics/back_date/" + post.id + "/" + dateTime, { type: 'POST' }).then(function (result) {
+          post.set('created_at', result.basic_post.created_at);
+          location.reload();
```

Comment:
I believe I commented on this last time you submitted this PR, but you shouldn't reload the page just because something changed. You should just update the object that changed and handlebars will update the UI automatically.